docs/discussion/implementation/applying_magnitudes.md

```diff
@@ -0,0 +1,136 @@
+# Applying Magnitudes
+
```

Comment:
This page is all about **Magnitudes**, but the word "magnitude" itself isn't explicitly stated or mentioned in the introduction -- we don't see the word again until the **Magnitude Categories** section.

Is it possible that this introduction should include the content in the **Magnitude Categories** section?

Reply:
Nice catch --- WDYT about 21db19d?

Reply:
That likely contextualizes the rest of the page well. Thank you!